// Generated by dev-tools; DO NOT EDIT.
// Modify or add entity types in proto/core/types/v1/asset_type.proto
// Modify template at dev-tools/contracts/templates/public/entity_types.proto.tpl
// Regenerate: `task regenerate-public-protos`

syntax = "proto3";
package synq.entities.v1;

option go_package = "github.com/getsynq/api/entities/v1";

enum EntityType {
  ENTITY_TYPE_UNSPECIFIED = 0;
  ENTITY_TYPE_BQ_TABLE = 103;
  ENTITY_TYPE_BQ_VIEW = 105;
  ENTITY_TYPE_LOOKER_LOOK = 201;
  ENTITY_TYPE_LOOKER_EXPLORE = 203;
  ENTITY_TYPE_LOOKER_VIEW = 207;
  ENTITY_TYPE_LOOKER_DASHBOARD = 208;
  ENTITY_TYPE_DBT_MODEL = 301;
  ENTITY_TYPE_DBT_TEST = 302;
  ENTITY_TYPE_DBT_SOURCE = 303;
  ENTITY_TYPE_DBT_PROJECT = 306;
  ENTITY_TYPE_DBT_METRIC = 307;
  ENTITY_TYPE_DBT_SNAPSHOT = 310;
  ENTITY_TYPE_DBT_SEED = 311;
  ENTITY_TYPE_DBT_ANALYSIS = 312;
  ENTITY_TYPE_DBT_EXPOSURE = 313;
  ENTITY_TYPE_DBT_GROUP = 314;
  ENTITY_TYPE_DBT_CLOUD_PROJECT = 352;
  ENTITY_TYPE_DBT_CLOUD_JOB = 353;
  ENTITY_TYPE_SNOWFLAKE_TABLE = 503;
  ENTITY_TYPE_SNOWFLAKE_VIEW = 508;
  ENTITY_TYPE_SNOWFLAKE_STREAM = 509;
  ENTITY_TYPE_SNOWFLAKE_DYNAMIC_TABLE = 510;
  ENTITY_TYPE_SNOWFLAKE_TASK = 511;
  ENTITY_TYPE_REDSHIFT_TABLE = 803;
  ENTITY_TYPE_REDSHIFT_VIEW = 805;
  ENTITY_TYPE_TABLEAU_EMBEDDED = 1101;
  ENTITY_TYPE_TABLEAU_PUBLISHED = 1102;
  ENTITY_TYPE_TABLEAU_CUSTOM_SQL = 1103;
  ENTITY_TYPE_TABLEAU_TABLE = 1104;
  ENTITY_TYPE_TABLEAU_SHEET = 1105;
  ENTITY_TYPE_TABLEAU_DASHBOARD = 1106;
  ENTITY_TYPE_AIRFLOW_DAG = 1201;
  ENTITY_TYPE_AIRFLOW_TASK = 1202;
  ENTITY_TYPE_CLICKHOUSE_TABLE = 1303;
  ENTITY_TYPE_CLICKHOUSE_VIEW = 1305;
  ENTITY_TYPE_ANOMALY_MONITOR = 1403;
  ENTITY_TYPE_ANOMALY_MONITOR_SEGMENT = 1404;
  ENTITY_TYPE_SQLTEST_TEST = 1421;
  ENTITY_TYPE_POSTGRES_TABLE = 1603;
  ENTITY_TYPE_POSTGRES_VIEW = 1605;
  ENTITY_TYPE_MYSQL_TABLE = 1703;
  ENTITY_TYPE_MYSQL_VIEW = 1705;
  ENTITY_TYPE_DATABRICKS_WAREHOUSE = 1801;
  ENTITY_TYPE_DATABRICKS_TABLE = 1804;
  ENTITY_TYPE_DATABRICKS_VIEW = 1805;
  ENTITY_TYPE_DATABRICKS_JOB = 1807;
  ENTITY_TYPE_DATABRICKS_NOTEBOOK = 1809;
  ENTITY_TYPE_DATABRICKS_QUERY = 1810;
  ENTITY_TYPE_DATABRICKS_DASHBOARD = 1811;
  ENTITY_TYPE_SQLMESH_PROJECT = 1901;
  ENTITY_TYPE_SQLMESH_SQL_MODEL = 1902;
  ENTITY_TYPE_SQLMESH_PYTHON_MODEL = 1903;
  ENTITY_TYPE_SQLMESH_EXTERNAL = 1904;
  ENTITY_TYPE_SQLMESH_SEED = 1905;
  ENTITY_TYPE_SQLMESH_AUDIT = 1906;
  ENTITY_TYPE_SQLMESH_UNIT_TEST = 1907;
  ENTITY_TYPE_SQLMESH_ENVIRONMENT = 1908;
  ENTITY_TYPE_SQLMESH_SNAPSHOT = 1909;
  ENTITY_TYPE_DUCKDB_TABLE = 2003;
  ENTITY_TYPE_DUCKDB_VIEW = 2005;
  ENTITY_TYPE_TRINO_TABLE = 2104;
  ENTITY_TYPE_TRINO_VIEW = 2105;
  ENTITY_TYPE_ATLAN_ASSET = 2201;
  ENTITY_TYPE_CUSTOM_ENTITY_GENERIC = 50000;
  ENTITY_TYPE_CUSTOM_ENTITY_CUSTOM_TYPE_MIN = 50001;
  ENTITY_TYPE_CUSTOM_ENTITY_CUSTOM_TYPE_MAX = 59999;
}
